### PR TITLE
mixer lint / fmt fixes

### PR DESCRIPTION
**Pull request by jmuk**
Not sure why these have been overlooked, but somehow they started
appearing on my local environment after updated to go 1.10rc2.

Fixes for other components will come as separate PRs.

### CODE REVIEW

File: mixer/pkg/runtime2/dispatcher/session.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ const queueAllocSize = 64
 
 // session represents a call session to the Dispatcher. It contains all the mutable state needed for handling the
 // call. It is used as temporary memory location to keep ephemeral state, thus avoiding garbage creation.
-type session struct {
+type session struct { // nolint: maligned
```
Comment (geeknoid) [inline mixer/pkg/runtime2/dispatcher/session.go]: This is a runtime data structure, we should fix this warning by rearranging the fields.

Comment (jmuk) [reply mixer/pkg/runtime2/dispatcher/session.go]: Okay... fixed.

**Comment by douglas-reid** [comment]
Did we upgrade the circle CI image to 1.10?  because some of these changes reflect formatting 1.10 choices.

**Comment by jmuk** [comment]
not sure about circle-ci -- and  note that 1.10 isn't officially released yet, it's still rc2. Seems 1.9.4 is released quite recently and that might affect.

**Pull request merged**